src/devices/Bmp180/samples/Bmp180.Sample.cs

```diff
@@ -30,23 +31,23 @@ static void Main(string[] args)
 
                 //read values
                 Temperature tempValue = i2cBmp280.ReadTemperature();
-                Console.WriteLine($"Temperature {tempValue.Celsius} \u00B0C");
+                Console.WriteLine($"Temperature: {tempValue.Celsius:0.#}\u00B0C");
                 var preValue = i2cBmp280.ReadPressure();
-                Console.WriteLine($"Pressure {preValue.Hectopascal} hPa");
-                double altValue = i2cBmp280.ReadAltitude();
-                Console.WriteLine($"Altitude {altValue:0.##} m");
+                Console.WriteLine($"Pressure: {preValue.Hectopascal:0.##}hPa");
+                var altValue = WeatherHelper.CalculateAltitude(preValue, Pressure.MeanSeaLevel, tempValue);

```

Comment:
Why would you want the sample to read the temperature and pressure twice instead of once? These sensors were intended for mobile application where energy usage is required to be very low.

Reply:
The samples are made to showcase the usage of the API of that device binding, which is why I want to restore the binding specific methods instead of using a completely different class here since that would defeat the purpose of the samples.

Reply:
You're asking to showcase a very inefficient way to achieve the same result though. I know this is not in line with general Microsoft sample applications I've seen. Up to you though, but I strongly disagree with your suggestion.

Reply:
That is why I thought to have the code for running it using the the bme sensor, but adding a comment above it with something like
```c#
// Note that if you already have the pressure value and the temperature, you could also calculate altitude by 
// calling WeatherHelper.CalculateAltitude(preValue, Pressure.MeanSeaLevel, tempValue) which would be more performant.
double altValue = i2cBmp280.ReadAltitude();
```

Reply:
That way, the samples are still showing how to consume this binding to perform operations, but if somebody cares about performance, they will read the comment and figure out that there is a faster way of doing it.

Reply:
Fair enough :-)

Reply:
Done (pending commit)